CHANGELOG.md

```diff
@@ -106,7 +106,17 @@ These changes are available on the `master` branch, but have not yet been releas
 - Fixed `ForumChannel.edit` allowing `default_reaction_emoji` to be `None`.
   ([#2739](https://github.com/Pycord-Development/pycord/pull/2739))
 - Fixed missing `None` type hints in `Select.__init__`.
-  ([#2746](https://github.com/Pycord-Development/pycord/pull/2746))
+  ([#2746])(https://github.com/Pycord-Development/pycord/pull/2746)
+- Fixed parameters for channels And allow creating channels with None as default
+- Added missing parameters to `create_text_channel`: `default_thread_slowmode_delay` and
+  `default_auto_archive_duration`.
+- Added missing parameters to `create_voice_channel`: `slowmode_delay` and `nsfw`.
+- Added missing parameters to `create_stage_channel`: `bitrate`, `user_limit`,
+  `rtc_region`, `video_quality_mode`, `slowmode_delay`, and `nsfw`.
+- Added missing parameters to `create_forum_channel`: `available_tags`,
+  `default_sort_order`, `default_thread_slowmode_delay`, and
+  `default_auto_archive_duration`. Also allow `default_reaction_emoji` to be `None`.
+  ([#2772])(https://github.com/Pycord-Development/pycord/pull/2772)
```

Comment:
This should be condensed into something like "Added various missing channel parameters." and should be moved to the proper category.

I don't know what `Fixed parameters for channels And allow creating channels with None as default` refers to.